_posts/2022-06-03-gaps-in-bitcoin-dev-funding.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "The gaps in bitcoin open-source funding"
+date:   2022-06-03 09:57:39 -0500
+categories: bitcoin funding grants
+---
+
+A few people reached out about my post on my [Guide for Grant Seekers](/bitcoin/funding/grants/grants-bitcoin-open-source/) with some critical feedback about promoting work for free culture. To clarify, this guide was not meant to be an ideal vision for the future, it was meant to reflect the present and help grant seekers achieve their goals.
+
+At the end of the post, I mentioned that this doesn't serve a lot of different groups. In fact, I think it serves a limited population which is why we see such geographic density of bitcoin FOSS devs. People that cannot do work for free over a long period aren't going to be able to make significant progress on becoming a FOSS dev quickly. Working on an all-consuming start-up, raising a family, or living in a place with limited electricity/internet will put grant seekers at a significant disadvantage. Because the proof of work is lower for other projects, this also means that, by default, they are more inclusive. I am biased because free work is such a useful screen. It's almost unforgeable proof of intrinsic motivation.
+
+## A funding bridge
+
+The big gap I see that isn't currently filled in the funding landscape is for people who can't quit their job or can only afford to work nights and weekends because of other commitments. Sure we see part-time grants, but building up one's credibility to get to that point takes investment. Funders aren't really taking a chance on people. The [residency](https://residency.chaincode.com/) filled this gap for some in the past. It was an almost-guarantee of finding FOSS funding. The residencies were never meant to scale and they are in hibernation anyways. Instead, we need a pot of cash to fund prospects. Let's call them bridge grants.
```

Comment:
```suggestion
The big gap I see that isn't currently filled in the funding landscape is for people who can't quit their job or can only afford to work nights and weekends because of other commitments. Sure we see part-time grants, but building up one's credibility to get to that point takes investment. Funders aren't really taking a chance on people. The [residency](https://residency.chaincode.com/) filled this gap for some in the past. It was an almost-guarantee of finding FOSS funding. The residencies were never meant to scale, and they are in hibernation anyways. Instead, we need a pot of cash to fund prospects. Let's call them bridge grants.
```